libs/api/repository/src/lib/gn4/platform/gn4-platform.mapper.ts

```diff
@@ -90,6 +90,12 @@ export class Gn4PlatformMapper {
           url: new URL(matchedThesaurus?.url),
           type: matchedThesaurus?.dname as KeywordType,
         },
+        coords: {
+          coordEast: keyword.coordEast,
```

Comment:
The coords are saved as strings. Should the parsing to a number already happen here?
And maybe already return a bbox instead?

Reply:
The way you did it is fine, I'm not sure we will need those coords as string anywhere else yet.

As for the order of the coords, were did you get it come from? In an object it doesn't really matter, the properties are not ordered, but later on I see you have to switch to arrays, and the order is really hard to follow.

In GIS, the most frequent order is [minx, miny, maxx, maxy] which is the equivalent of [westBound, southBound, eastBound, northBound].